app/main/views/frameworks.py

```diff
@@ -66,20 +76,26 @@ def framework_services():
 @flask_featureflags.is_active_feature('GCLOUD7_OPEN')
```

Comment:
Marginally as both render the same template. I'm not very opinionated either way but we should be consistent. Maybe something for the tech debt wall for us to decide and change wholesale?